app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="log-in">
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
```

Comment:
Please make sure that a user can register before trying to login into the app. The user gets the `Login` page every time while trying to register. 

Please make user that the application asks for confirmation by email.

Reply:
Done

Reply:
done